config/environments/production.rb

```diff
@@ -44,6 +44,13 @@
   # Allow to specify public IP of reverse proxy if it's needed
   config.action_dispatch.trusted_proxies = ENV['TRUSTED_PROXY_IP'].split.map { |item| IPAddr.new(item) } if ENV['TRUSTED_PROXY_IP'].present?
 
+  config.force_ssl = true
```

Comment:
I remember there were some recent Tor-related changes that mentioned force_ssl, does this conflict with that?